charts/generic-prometheus-alerts/ci/tests/ingress-alerts-tests.yaml

```diff
@@ -8,9 +8,9 @@ evaluation_interval: 1m
 tests:
   - interval: 1m
     input_series:
-      - series: 'nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path="/bob", status="500"}'
```

Comment:
these tests were wrong previously - there is only the `/` and `/health` endpoint in grafana / prometheus